Singularity/Shaders/SingularityShaders/Assets/Shaders/BlackHoleAccretionDisk.shader

```diff
@@ -286,7 +287,8 @@
 			  		// 0.05: rate of smaller steps when approaching blackhole
 			  		stepSize = rayDistance - (blackHoleRadius *0.05);
 
-			  		rayAccel = normalize(gravityVector) * gravity / (rayDistance * rayDistance);
+			  		//rayAccel = normalize(gravityVector) * gravity / (rayDistance * rayDistance);
+					rayAccel = normalize(gravityVector) * schwarzschildRadius*schwarzschildRadius/1.04976E9 / (rayDistance * rayDistance);
```

Comment:
I'd suggest precomputing this value before passing it to the shader.

Also there is now blackHoleRadius and schwarzschildRadius, what's the difference between both? Can one be expressed in terms of the other?

Also remove the gravity parameter if it's no longer used

Reply:
In v0.994, I switched back to `gravity` in the shader. Now `gravity` can be pre-computed using `schwarzschildRadius`.

`blackHoleRadius` is used in the original code. It's the scaled space radius of the black hole and equals to schwarzschildRadius/6000. `schwarzschildRadius` functions as an alternative to `gravity`, which equals the actual Schwarzschild radius in meters.

Reply:
Makes sense, I just didn't want there to be multiple radius variables in the shader code. If the difference is just being divided by 6000 that just means the blackHoleRadius is in scaledSpace units and schwarzschildRadius is in local space units but they are basically the same thing.